docs/pages/reference/runtimes.md

```diff
@@ -0,0 +1,116 @@
+# Runtimes
```

Comment:
Is this a good term for it? I am also considering "Executors", "Engines", ...

Reply:
Of the three, I prefer `Runtimes`, just like you ahve it now :)